controllers/useraccount/useraccount_controller.go

```diff
@@ -484,6 +500,93 @@ func (r *Reconciler) deleteUser(ctx context.Context, userAcc *toolchainv1alpha1.
 	return true, nil
 }
 
+// Returns `true` if the associated resources (configMap, role and role-binding) created for a user by console is deleted, `false` otherwise with the underlying error.
+// This function only looks for these resources in the namespace - openshift-console-user-settings -
+func (r *Reconciler) deleteUserResources(ctx context.Context, userID string) (bool, error) {
+
+	// new user will have a label introduced which can help retrieve resources, otherwise older ones will have to rely on the name of the resource being of type `user-settings-<UID>`
+	// delete ConfigMap, Role and RoleBinding
```

Comment:
```suggestion
	// Users which were created in the cluster with the OCP versions which includes https://issues.redhat.com/browse/OCPBUGS-32321 fix
	// will have a label which will help to map the User to the User settings resources.
	// User which were created before that won't have that label and we have to rely on the name of the resource being of type `user-settings-<UID>`,
	// where <UID> is the User UID.

	// Delete ConfigMap, Role and RoleBinding
```

Reply:
done in [1bf005e](https://github.com/codeready-toolchain/member-operator/pull/568/commits/1bf005eeb31a8b6968271c667f5b5777c5f4c969)